config/locales/en/brexit_checker/action_audiences.yml

```diff
@@ -1,4 +1,4 @@
 en:
   brexit_checker:
     action_audiences:
-      no_results: This is a safe and secure service. We don't store any of your personal information.
```

Comment:
Is it intentional that we've removed this messaging around storing of personal information?

Reply:
Yeah I believe so, the doc said replace this where the results are shown. Will confirm with content though - i agree its weird.